src/SourceBuild/content/.devcontainer/prebuilt-sdk/build.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+./prep.sh
+
+# GitHub Codespaces automatically sets RepositoryName, which conflicts with source-build scripts.
+unset RepositoryName
+
+./build.sh --online --clean-while-building || true
```

Comment:
Also, the `|| true` at the end to force exit 0 might be more readable by adding `exit 0` at the end to state the intent.